e2e/commands/start/start_test.go

```diff
@@ -18,36 +18,33 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/ZupIT/horusec-devkit/pkg/utils/logger/enums"
+	"github.com/ZupIT/horusec/internal/enums/outputtype"
+
 	"github.com/google/uuid"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gexec"
 
-	"github.com/ZupIT/horusec-devkit/pkg/utils/logger/enums"
-
 	"path/filepath"
 
 	"github.com/ZupIT/horusec/internal/utils/testutil"
 )
 
 var _ = Describe("running binary Horusec with start parameter", func() {
 	var (
-		session           *gexec.Session
-		err               error
-		flags             map[string]string
-		repoAuthorization string
-		configFilePath    = testutil.GoExample1
+		session        *gexec.Session
+		flags          map[string]string
+		configFilePath = testutil.GoExample1
 	)
 
 	JustBeforeEach(func() {
+		var err error
```

Comment:
this error can't be started in `session, err := gexec.Start(cmd, GinkgoWriter, GinkgoWriter)`?

Reply:
do i need the session to be "global", any suggestion?